### PR TITLE
[NUI] Fix CA2000 warning issues of View and ViewImpl.

### PR DESCRIPTION
**Pull request by AchoWang**
### Description of Change ###
Changed files:
src/Tizen.NUI/src/internal/ViewImpl.cs
src/Tizen.NUI/src/public/BaseComponents/View.cs

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by zhouleonlei — APPROVED** [review]
LGTM

**Review by zhouleonlei — APPROVED** [review]
LGTM

**Review by zg2nets — APPROVED** [review]
looks good to me！

**Review by huayongxu — APPROVED** [review]
lgtm

**Review by elishateng — APPROVED** [review]
LGTM

File: src/Tizen.NUI/src/public/BaseComponents/View.cs [hunk 1]
```diff
@@ -908,9 +911,15 @@ public Vector2 ScreenPosition
...
             {
-                Vector2 temp = new Vector2(0.0f, 0.0f);
-                GetProperty(View.Property.ScreenPosition).Get(temp);
-                return temp;
+                using (PropertyValue screenPosition = GetProperty(View.Property.ScreenPosition)) 
+                {
+                    using (Vector2 vectorValue = new Vector2(0.0f, 0.0f))
+                    {
+                        Vector2 position = vectorValue;
+                        screenPosition?.Get(position);
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/public/BaseComponents/View.cs:919]: Although null check is cheap, but it does not necessary here compared with previous code.
Ditto to other APIs.

File: src/Tizen.NUI/src/public/BaseComponents/View.cs [hunk 2]
```diff
@@ -1896,8 +1946,11 @@ public new View Parent
 
                 if (basehandle is Layer layer)
                 {
-                    ret = new View(Layer.getCPtr(layer).Handle, false);
-                    NUILog.Error("This Parent property is deprecated, shoud do not be used");
+                    using (View view = new View(Layer.getCPtr(layer).Handle, false))
+                    {
+                        ret = view;
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/public/BaseComponents/View.cs:1951]: This does not make sense.
When it gets out of the `using` scope, the`ret` value will be disposed.
It need to be fixed.

**Review by rabbitfor — COMMENTED** [review]
@AchoWang ,
Please check that the properties are not used after disposed.

**Pull request merged**